sonar-project.properties

```diff
@@ -0,0 +1,3 @@
+sonar.projectKey=omise_omise-java
+sonar.organization=omise
+systemProp.sonar.host.url=https://sonarcloud.io
```

Comment:
Do we need this file since these properties already define in here?
https://github.com/omise/omise-java/pull/154/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R158